src/CMakeLists.txt

```diff
@@ -27,3 +28,166 @@ add_library(bitcoin_consensus OBJECT EXCLUDE_FROM_ALL
   util/strencodings.cpp
 )
 target_link_libraries(bitcoin_consensus PRIVATE secp256k1)
+
+
+# Home for common functionality shared by different executables and libraries.
+# Similar to `bitcoin_util` library, but higher-level.
+add_library(bitcoin_common STATIC EXCLUDE_FROM_ALL
+  base58.cpp
+  bech32.cpp
+  chainparams.cpp
+  coins.cpp
+  common/bloom.cpp
+  common/interfaces.cpp
+  common/run_command.cpp
+  $<$<TARGET_EXISTS:PkgConfig::libevent>:common/url.cpp>
```

Comment:
This seems like a sign that something is broken. But sure enough, it exists in autotools.

I really don't like the idea of helper libs with conditional functionality. Obviously this is fine for now because it matches what we currently have, but I'd like to fix this up later.